src/foraging_gui/Dialogs.py

```diff
@@ -1316,8 +1312,7 @@ def _start_preview(self):
             self.MainWindow.Channel.CameraControl(int(1))
 
             self.StartPreview.setStyleSheet("background-color : green;")
-            self.WarningLabelCameraOn.setText('Camera is on')
```

Comment:
![image](https://github.com/user-attachments/assets/c23bea51-49c4-47c2-abd4-f4d245067a57)

Reply:
again, duplicate this one